config/groovy/module.groovy

```diff
@@ -79,6 +95,10 @@ class module {
 
     def refreshGradle(File targetDir) {
         // Copy in the template build.gradle for modules
+        if (!new File(targetDir, "module.txt").exists()) {
+            println "$targetDir has no module.txt, it must not want a fresh build.gradle"
```

Comment:
Did a spare "not" sneak in here?

Looking at this, I'd like to go over this and allow for `module.json` as alternative to `.txt`...

Reply:
If not module.txt, then no new build.gradle.

I don't see a different number of "not"s than I intended.

Are there directories without modules.txt we do want to refresh as gradle subprojects?

What I was trying to do was address the situation where something was making stray directories in there that weren't actually populated with module sources, and then they'd get gradle files, and then the gradle files would do the whole Y U NO HAVE MODULE.TXT yelling thing.